test/sharness/t0116-prometheus-data/prometheus_metrics

```diff
@@ -1,3 +1,5 @@
+active_block_tasks
+active_tasks
```

Comment:
I'd expect all these metrics to be prepended with bitswap_ as the other systems are